.travis.yml

```diff
@@ -7,4 +7,6 @@ script:
 - bin/fetch-configlet
 - bin/configlet lint .
 - make test-travis
+install:
+- npm install --no-save
```

Comment:
Just one nitpick, but would you mind moving the `install` block before `script`? I find it more appropriate in order:
* language
* node version
* install
* script

Reply:
Oops...I missed this comment.  I will move it, no problem at all.